packages/landing/src/config.map.ts

```diff
@@ -413,13 +413,22 @@ function addDefaultLayers(output: Map<string, LayerInfo>): void {
 
     {
       id: 'elevation',
-      title: 'Elevation',
+      title: 'Elevation DEM',
       projections: new Set([EpsgCode.Google]),
       category: 'Basemaps',
       pipeline: 'terrain-rgb',
       imageFormat: 'png',
     },
 
+    {
+      id: 'elevation-dsm',
+      title: 'Elevation DSM',
+      projections: new Set([EpsgCode.Nztm2000, EpsgCode.Google]),
```

Comment:
As discussed this morning. I think we want to replicate this with Elevation, and only add to webmercator projection.